### PR TITLE
Routes api

### PR DESCRIPTION
**Pull request by tinku-bansal**

### CODE REVIEW

File: routes/api/gameRoutes.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const { Op } = require("sequelize");
+
+const router = require("express").Router();
+
+// create new game
+router.post("/", async (req, res) => {
+  const { gameTitle } = req.body;
+
+  if (
+    ![gameTitle].every(
```
Comment (strahinjaajvaz) [inline routes/api/gameRoutes.js]: no point in doing it this way as it's a single value.

this should be fine.
```suggestion
    if(!gameTitle.trim()) {}
```

File: routes/api/gameRoutes.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    res.status(404).send({ message: "Invalid parameters" });
+    return;
+  }
+
+  const gameTitleValidate = await Games.findOne({
+    where: { gameTitle },
+  });
+
+  if (gameTitleValidate) {
+    res.status(404).json({ message: "A game with this title already exists!" });
```
Comment (strahinjaajvaz) [inline routes/api/gameRoutes.js]:
```suggestion
    res.status(404).json({ message: `A game with ${gameTitle} already exists!`});
```

**Review by strahinjaajvaz — CHANGES_REQUESTED** [review]
Awesome work. 
Please fix the following issues :)

**Comment by strahinjaajvaz** [comment]
Please resolve the merge conflicts also.

File: routes/api/gameRoutes.js (hunk 1 above)
Comment (tinku-bansal) [reply routes/api/gameRoutes.js]: Yeh, sorry my bad!

Comment (tinku-bansal) [reply routes/api/gameRoutes.js]: fixing it now

File: routes/api/gameRoutes.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ router.post("/", async (req, res) => {
   const { gameTitle } = req.body;
 
   if (
-    ![gameTitle].every(
+    if(!gameTitle.trim()) {}
       (item) => item.trim() !== ""
```
Comment (strahinjaajvaz) [inline routes/api/gameRoutes.js]: what is item?

Comment (tinku-bansal) [reply routes/api/gameRoutes.js]: fixed that!

**Review by KorolyovaLara — APPROVED** [review]
Thank you Tinku, I will combine it into the one we already have for games.

**Comment by tinku-bansal** [comment]
Thanks Lara!

**Pull request merged**